packages/sdk/src/job.ts

```diff
@@ -1,7 +1,7 @@
-import { get, getFile, del, post, FileDownloadResp } from './request';
+import { get, getFile, del, post } from './request';
 import { BaseApi } from './base';
-import { JobResp, TraceResp, JobListFilter } from './interface';
-
+import { JobResp, TraceResp, JobListFilter, FileDownloadResp } from './interface';
+import * as ContentDisposition from 'content-disposition';
```

Comment:
Shall we directly import `parse` from 'content-disposition'?

Reply:
Yep, it's better.